src/slider/styled-components.js

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import {styled} from '../styles/index';
+import {} from './constants';
+import {getToggleCheckMarkStyles} from '../checkbox/styled-components';
```

Comment:
This really may be a situation where it's better to copy the styles rather than trying to be overly DRY. Even though this looks similar to the toggle, we're now significantly complicating our checkbox logic by adding `$isStart`, `$isEnd`, etc.

Reply:
ok, removed extra code from `checkbox` and left it in slider so far